footer.html

```diff
@@ -2,11 +2,13 @@
 	<div class="kat-footer__contact-wrapper">
 		<div class="o-footer__container">
 			<div class="kat-footer__contact-container">
-				<div class="kat-footer__contact-item">Any questions? Contact Us</div>
-				<div class="kat-footer__contact-item">
-					<a data-trackable="contact-phone-uk" href="tel: +44 (0)20 7873 4001" data-original-title title>+44 (0)20 7873 4001</a>
-					or
-					<a data-trackable="contact-phone-us" href="tel: +1 877 843 3399" data-original-title title>+1 877 843 3399</a>
+				<div class="kat-footer__contact-item">Any questions? </div>
+				<div class="kat-footer__contact-item"> Call UK
+					<a class="o-typography-link" data-trackable="contact-phone-uk" href="tel: +44 (0)20 7873 4001" data-original-title title>+44 (0)20 7873 4001,</a>
+					Americas (toll free)
+					<a class="o-typography-link" data-trackable="contact-phone-us" href="tel: +1 877 843 3399" data-original-title title>+1 877 843 3399</a>
+					or Asia
+					<a class="o-typography-link" data-trackable="contact-phone-us" href="tel: +1 877 843 3399" data-original-title title>+632 982 5780</a>
```

Comment:
Need to update the `data-trackable` to `contact-phone-asia` for this one

Reply:
Well spotted, thanks!